build.gradle

```
@@ -82,6 +82,19 @@ allprojects {
     }
 }
 
+configure(allprojects
+        - project("arrow-kio-benchmarks")
+        - project("arrow-meta")
+        - project("arrow-ank-gradle")
+        - project("arrow-ank")
+) {
+    apply plugin: 'java'
+    apply plugin: 'ru.vyarus.animalsniffer'
+    dependencies {
+        signature 'net.sf.androidscents.signature:android-api-level-14:4.0_r4@signature'
```

```suggestion
        signature 'net.sf.androidscents.signature:android-api-level-21:5.0.1_r2@signature'
```

@junron Would you mind trying API 21 and see if everything is ok?